convert_jendl.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
+import argparse
+import glob
+import os
+import sys
+import tarfile
+import zipfile
+from collections import defaultdict
+from string import digits
+from urllib.request import urlopen
```

Comment:
A bunch of these imports are not used: zipfile, defaultdict, digits, urlopen

Reply:
Thanks for this, I noticed a few others in some of the convert scripts so I have also tidied them up